### PR TITLE
Fix arm, base, camera snippets

### PR DESCRIPTION
**Pull request by JessamyT**
DOCS-2779

### CODE REVIEW

File: src/viam/components/camera/camera.py
Comment (npentrel) [inline src/viam/components/camera/camera.py:52]: They can add it anywhere above where it's used. If they use python, they don't need to be told the rest
```suggestion
           from viam.media.video import CameraMimeType
```

**Review by npentrel — APPROVED** [review]
LGTM % nits

File: src/viam/components/camera/camera.py
Comment (npentrel) [inline src/viam/components/camera/camera.py:53]: Make sure to set your editor to trim whitespace at the end of lines
```suggestion

```

File: src/viam/components/camera/camera.py
Comment (npentrel) [inline src/viam/components/camera/camera.py:54]: If you haven't, make sure to set your editor to trim whitespace at the end of lines

```suggestion

            my_camera = Camera.from_robot(robot=machine, name="my_camera")
```

File: src/viam/components/camera/camera.py
Comment (purplenicole730) [inline src/viam/components/camera/camera.py:83]: Technically this should be `image_list = images`. The `[0]` means it's only the first image.

**Review by purplenicole730 — APPROVED** [review]
One comment and after Naomi's comments get resolved, LGTM

**Review by stuqdog — COMMENTED** [review]
lgtm modulo other folks' comments re: `image_list` and whitespace fixes.

File: src/viam/components/camera/camera.py
Comment (JessamyT) [reply src/viam/components/camera/camera.py:52]: Okay I agree, I was just doing this because that's how we do it in another docs-side example. I'll change that one too

File: src/viam/components/camera/camera.py
Comment (JessamyT) [reply src/viam/components/camera/camera.py:83]: I think 0 is the list of images and 1 is the metadata, according to the docs?

Comment (JessamyT) [reply src/viam/components/camera/camera.py:83]: My impression was that the first image would be images[0][0]?

Comment (purplenicole730) [reply src/viam/components/camera/camera.py:82]: If you look above, it's already split out. `images, metadata = await get_images...`, so it's already done the first [0].

Comment (JessamyT) [reply src/viam/components/camera/camera.py:82]:
```suggestion
            first_image = images[0]
```

Comment (JessamyT) [reply src/viam/components/camera/camera.py]: oOoh ok ty

**Pull request merged**
